src/ui/components/pages/Account/Account.tsx

```diff
@@ -31,7 +33,18 @@ export function Account(props: Props) {
 
     const { t } = useTranslation({ Account });
 
-    const tabs = useMemo(() => accountTabIds.map(id => ({ id, "title": t(id) })), [t]);
+    const availableDeploymentRegions = useSelector(
+        state => state.deploymentRegion.availableDeploymentRegions,
+    );
+    const kubernetesServerUrl = availableDeploymentRegions[0].kubernetes?.url || "";
+
+    const tabs = useMemo(
+        () =>
+            accountTabIds
+                .filter(name => name !== "kubernetes" || kubernetesServerUrl)
+                .map(id => ({ id, "title": t(id) })),
+        [t],
+    );
```

Comment:
Is there a need for a special case for the new tab?

Reply:
Ref here : https://github.com/InseeFrLab/onyxia-web/issues/272#issuecomment-979297649 and here https://github.com/InseeFrLab/onyxia-web/issues/272#issuecomment-1240417262